### PR TITLE
Feature/avatar&badge

### PR DESCRIPTION
**Pull request by Nikhil-Nirmal**

### CODE REVIEW

File: components/Badge/Badge.css [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+.badge .num {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    top : -.2rem;
+    right: -.1rem;
+    position: absolute;
+    font-size: .8rem;
+    color: #ffffff;
+    background-color: #5cc3ff;
```
Comment (shraddha15-creator) [inline components/Badge/Badge.css:45]: why not using color variables everywhere?

**Review by shraddha15-creator — APPROVED** [review]
Use color variables. Rest code looks good to me

File: components/Badge/Badge.css (hunk 1 above)
Comment (Nikhil-Nirmal) [reply components/Badge/Badge.css:45]: Updated with color variables.

File: components/Avatar/Avatar.css [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    Author Name:Nikhil Nirmal,
+    color pallet used:
+    background:#0a192f;
+    font 1 : #55dabe;
+    font 2 : #ccd6f6;
+    font 3 : #6e7996;   
+*/
+
+.avatar-sm {
+    width: 30px;
```
Comment (rutvikpumak) [inline components/Avatar/Avatar.css]: convert px to rem everywhere which is more than 15 or 20px

File: components/Avatar/Avatar.html [hunk 3]
```diff
@@ -0,0 +1,220 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="Avatar.css">
+    <link rel="stylesheet" href="../../Css/open.css">
+    <link rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.4.0/styles/tomorrow-night-blue.min.css">
+    <title>Document</title>
```
Comment (rutvikpumak) [inline components/Avatar/Avatar.html]: title should be avatar

File: components/Avatar/Avatar.html [hunk 4]
```diff
@@ -0,0 +1,220 @@
...
+            document.getElementsByClassName("errorAlert")[0].classList.add("dismiss")
+        }
+    </script>
+    <script>
+        function openNav() {
+          document.getElementById("mySidenav").style.width = "250px";
+        }
+        
+        function closeNav() {
+          document.getElementById("mySidenav").style.width = "0";
```
Comment (rutvikpumak) [inline components/Avatar/Avatar.html:214]: add js part to js file

File: components/Badge/Badge.css [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+.on {
+    background-color: greenyellow;
+}
+
+.off {
+    background-color: #757575;
+}
+
+.dnd {
```
Comment (rutvikpumak) [inline components/Badge/Badge.css:33]: use variable for colors

File: components/Badge/Badge.html [hunk 6]
```diff
@@ -0,0 +1,234 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="Badge.css">
+    <link rel="stylesheet" href="../Avatar/Avatar.css">
+    <link rel="stylesheet" href="../../Css/open.css">
+    <link rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.4.0/styles/tomorrow-night-blue.min.css">
+    <title>Document</title>
```
Comment (rutvikpumak) [inline components/Badge/Badge.html]: title to badge

File: components/Badge/Badge.html [hunk 7]
```diff
@@ -0,0 +1,234 @@
...
+        function dismissErrorAlert() {
+            document.getElementsByClassName("errorAlert")[0].classList.add("dismiss")
+        }
+    </script>
+    <script>
+        function openNav() {
+          document.getElementById("mySidenav").style.width = "250px";
+        }
+        
+        function closeNav() {
```
Comment (rutvikpumak) [inline components/Badge/Badge.html:227]: move to separate js file

**Review by rutvikpumak — APPROVED** [review]

**Pull request merged**